tools/topology/topology2/include/common/mod_cfg.conf

```diff
@@ -0,0 +1,45 @@
+# Class definition for module configuration object
+# mod_cfg objects can be instantiated as:
+#
+# Object.Base.mod_cfg."0" {
+#	ibs	384
+#	obs	384
+#	cpc	4600
```

Comment:
can we add comments here:
for ibs/obs, it is initialized as: 48 * 2 * 4 = 384, it is 48k LL with 2 channel 32 bit format.
for cpc, I am not sure how CPC figured out as instantiated value?